tests/testExposure.py

```diff
@@ -154,6 +154,35 @@ def testGetMaskedImage(self):
         self.assertEqual(crOnlyWidth, self.exposureCrOnly.getWidth())
         self.assertEqual(crOnlyHeight, self.exposureCrOnly.getHeight())
 
+
+    def testProperties(self):
+        self.assertMaskedImagesEqual(self.exposureMiOnly.maskedImage,
+                                     self.exposureMiOnly.getMaskedImage())
+        mi2 = afwImage.MaskedImageF(self.exposureMiOnly.getDimensions())
+        mi2.image.array = 5.0
+        mi2.variance.array = 3.0
+        mi2.mask.array = 0x1
+        self.exposureMiOnly.maskedImage = mi2
+        self.assertMaskedImagesEqual(self.exposureMiOnly.maskedImage, mi2)
+        self.assertImagesEqual(self.exposureMiOnly.image,
+                               self.exposureMiOnly.maskedImage.image)
+
+        image3 = afwImage.ImageF(self.exposureMiOnly.getDimensions())
+        image3.array = 3.0
+        self.exposureMiOnly.image = image3
+        self.assertImagesEqual(self.exposureMiOnly.image, image3)
+
+        mask3 = afwImage.MaskU(self.exposureMiOnly.getDimensions())
+        mask3.array = 0x2
+        self.exposureMiOnly.mask = mask3
+        self.assertMasksEqual(self.exposureMiOnly.mask, mask3)
+
+        var3 = afwImage.ImageF(self.exposureMiOnly.getDimensions())
+        var3.array = 2.0
+        self.exposureMiOnly.variance = var3
+        self.assertImagesEqual(self.exposureMiOnly.variance, var3)
+
+
```

Comment:
I would strongly suggest numpy-like behavior, so `x.array = scalar` means `x.array` is a scalar (or, in this case, perhaps raises, since we should probably enforce `image`, `mask`, and `variance` having the same shape?), but `x.array[:] = scalar` sets the whole array to `scalar`, and similarly `x.array += scalar` should work as expected.